client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -371,162 +396,160 @@ export default function CampaignItemDetails( props: Props ) {
+												</span>
+												<span className="campaign-item-details__text wp-brand-font">
+													{ ! isLoading ? clicks_total : <FlexibleSkeleton /> }
+												</span>
+											</div>
+											<div>
+												<span className="campaign-item-details__label">
+													{ translate( 'Conversion Rate' ) }
+												</span>
+												<span className="campaign-item-details__text wp-brand-font">
+													{ ! isLoading ? ctrFormatted : <FlexibleSkeleton /> }
+												</span>
+											</div>
+										</div>
+									) }
 								</div>
-								<div className="campaign-item-details__main-stats-row">
+							</div>
+						) }
+						<div className="campaign-item-details__main-stats-container">
+							<div className="campaign-item-details__secondary-stats">
+								<div className="campaign-item-details__secondary-stats-row">
 									<div>
 										<span className="campaign-item-details__label">
-											{ translate( 'Duration' ) }
+											{ translate( 'Weekly Budget' ) }
 										</span>
 										<span className="campaign-item-details__text wp-brand-font">
-											{ ! isLoading ? durationFormatted : <FlexibleSkeleton /> }
+											{ ! isLoading ? totalBudgetFormatted : <FlexibleSkeleton /> }
 										</span>
 										<span className="campaign-item-details__details">
-											{ ! isLoading ? (
-												`${ duration_days } ${ translate( 'days' ) }`
+											{ ! isLoading && dailyAverageSpending ? (
+												/* translators: Daily average spend: the current budget divided by the amount of days of the campaign */
+												`${ translate( 'Daily av. spend' ) } $${ dailyAverageSpending }`
 											) : (
 												<FlexibleSkeleton />
 											) }
 										</span>
 									</div>
-									<div>
-										<span className="campaign-item-details__label">{ translate( 'Budget' ) }</span>
-										<span className="campaign-item-details__text wp-brand-font">
-											{ ! isLoading ? totalBudgetFormatted : <FlexibleSkeleton /> }
-										</span>
-										<span className="campaign-item-details__details">
-											{ ! isLoading ? totalBudgetLeftFormatted : <FlexibleSkeleton /> }
-										</span>
-									</div>
-									<div>
-										<span className="campaign-item-details__label">
-											{ translate( 'Overall spending ' ) }
-										</span>
-										<span className="campaign-item-details__text wp-brand-font">
-											{ ! isLoading ? overallSpendingFormatted : <FlexibleSkeleton /> }
-										</span>
-									</div>
-								</div>
-							</div>
-						</div>
-						<div className="campaign-item-details__main-stats-container">
-							<div className="campaign-item-details__secondary-stats">
-								<div className="campaign-item-details__secondary-stats-row">
 									<div>
 										<span className="campaign-item-details__label">
-											{ translate( 'Estimated impressions' ) }
+											{ translate( 'Weekly impressions' ) }
```

Comment:
corrected

Reply:
Seeing the final solution, I think it gets repetitive to show estimated impressions twice. It will be better to remove the subtitle for this iteration.

![Screenshot 2024-02-23 at 10 53 49](https://github.com/Automattic/wp-calypso/assets/1258162/8f121134-32dc-41a5-8673-a54ce8854661)